src/composables/multisigAccounts.ts

```diff
@@ -93,13 +94,14 @@ export function useMultisigAccounts({ store }: UseMultisigAccountsOptions) {
   }
 
   /**
+   * Prepare multisig account creation transaction
    * @param {number} noOfConfirmations no of minimum confirmation needed to process a transaction
    * @param {string[]} signers
    */
-  async function deployMultisigAccount(
+  async function prepareMultisigCreationTx(
```

Comment:
I suggest to rename this func to `prepareMultisigAccountCreationTx` (add "Account" as there is no such a thing as just "Multisig").